tests/main_tests/median_test.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import unittest
+
+import pandas as pd
+
+sys.path.append("tests")
+sys.path.append("..src")
+from src.utils.helper_functions import find_median
+
+
+class TestGetMedian(unittest.TestCase):
```

Comment:
Minor comment: consider matching the name of the function which is tested here, i.e. `find` vs `get`.